feature/networkinstance/otg_tests/defaults_test/defaults_test.go

```diff
@@ -38,7 +38,15 @@ func TestMain(m *testing.M) {
 
 func assignPort(t *testing.T, d *oc.Root, intf, niName string, a *attrs.Attributes) {
 	t.Helper()
-	d.GetOrCreateNetworkInstance(niName)
+	ni := d.GetOrCreateNetworkInstance(niName)
+	if niName != *deviations.DefaultNetworkInstance {
+		ni.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
+	}
+	if niName != *deviations.DefaultNetworkInstance || *deviations.ExplicitInterfaceInDefaultVRF {
```

Comment:
In this if condition, shouldn't we add the interfaces into the VRF? 
```
		fptest.AssignToNetworkInstance(t, dut, dut.Port(t, "port1").Name(), *deviations.DefaultNetworkInstance, 0)
		fptest.AssignToNetworkInstance(t, dut, dut.Port(t, "port2").Name(), *deviations.DefaultNetworkInstance, 0)
	}
```

Reply:
That is what the following codes do.
```
		niIntf := ni.GetOrCreateInterface(intf)
		niIntf.Interface = ygot.String(intf)
		niIntf.Subinterface = ygot.Uint32(0)
```
Don't want to use AssignToNetworkInstance here as it will push the config to device right away. assignPort func is just to build up the oc data.